api/v1/storagecluster_types.go

```diff
@@ -251,10 +251,12 @@ type ManageCephRBDMirror struct {
 }
 
 // MgrSpec defines the settings for the Ceph Manager
+// +structType:=atomic
```

Comment:
what does this do?

Reply:
This basically resets any earlier set value in the struct, So in this case let's say someone had EnableActivePassive as true, Now they mark DisableActivePassive as true, So the earlier flag automatically gets reset.

Ref-https://book.kubebuilder.io/reference/markers/crd-processing#:~:text=string-,specifies%20the%20level%20of%20atomicity%20of%20the%20struct%3B%20i.e.%20whether%20each%20field%20in%20the%20struct%20is%20independent%20of%20the%20others%2C%20or%20all%20fields%20are%20treated%20as%20a%20single%20unit.,-Possible%20values%3A